React/Recipes/Caching NextJS public folder.md

```diff
@@ -0,0 +1,102 @@
+## Motivation
+
+The `Cache-Control` HTTP header allows setting the caching for the resources used by the browser.
+It is always a good practice to check if some resources can be cached, so the user's browser can use them from the cache 
+without consuming more data and without waiting on the resource to finish downloading.
```

Comment:
Do not split sentences into multiple lines. Leave it to the markup and styling to take care of that. This occurs in a few places below as well.